rpcs3/Emu/system_config.h

```diff
@@ -250,6 +250,7 @@ struct cfg_root : cfg::node
 		cfg::_enum<CellSysutilLang> language{ this, "Language", CellSysutilLang{1} }; // CELL_SYSUTIL_LANG_ENGLISH_US
 		cfg::_enum<CellKbMappingType> keyboard_type{ this, "Keyboard Type", CellKbMappingType{0} }; // CELL_KB_MAPPING_101 = US
 		cfg::_enum<enter_button_assign> enter_button_assignment{ this, "Enter button assignment", enter_button_assign::cross };
+		cfg::_int<-60*60*24*365*100, 60*60*24*365*100> console_time_offset{ this, "Console time offset (s)", 0 }; //console time offset, limited to +/-100years
```

Comment:
I don't see these min and max being handled in the widget.
EnhanceDateTimeEdit needs to setDateRange once on setup and then setDateRange again on each update in the running timer